libs/utils/utils.js

```diff
@@ -1211,8 +1229,6 @@ export async function loadArea(area = document) {
   }
   const config = getConfig();
 
-  await decoratePlaceholders(area, config);
```

Comment:
`loadArea` is used by a bunch of other [logic](https://github.com/search?q=org%3Aadobecom+loadarea&type=code). Are we sure this won't lead to placeholders not being decorated on other projects making use of the `loadArea` method?

Reply:
`loadArea` still calls `processSection` which in turn calls `decoratePlaceholders` 

So the call moved into an embedded function of `loadArea` - Hence, I don't think this will lead to issues